build.gradle

```diff
@@ -1,9 +1,26 @@
- allprojects  {
-   apply plugin: 'maven'
+allprojects {
+    apply plugin: 'groovy'
+    apply plugin: 'maven'
+    apply plugin: 'build-dashboard'
 }
+
 subprojects {
-  repositories {
-    mavenLocal()
-    mavenCentral()
-  }
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+
+    dependencies {
```

Comment:
you'll still want this section in your `libera/build.gradle` since you will end up having only a few deps in common with me i think. also i don't think you are using jyaml anywhere at the moment.